src/frontend/Player/createPlayer.ts

```diff
@@ -17,3 +17,5 @@ export const createPlayer = (
       return createPlyrPlayer(ref, jwt, dispatch);
   }
 };
+
+export type PlayerCreator = typeof createPlayer;
```

Comment:
this should be in the `type` folder ?

Reply:
You also have a typo in the commit body, `funciton` => `function`

Reply:
IMO the type folder should not depend on non-type stuff. In this case I should probably define `PlayerCreator` separately and just ensure createPlayer matches it

Reply:
ack

Reply:
Done